### PR TITLE
Refactor database, fixes #95

### LINKED ISSUE #95: Change relations among the models
After refactoring of the player model: member, player, address, contact are related this the user. 
After a while this relation feel not logically. And I plan the following change.

img_20160829_094122

This may make it possible to change the routes(idea: @elgrillo83)
we could nest `:addresses` under `:members`, `:events`, `:teams`.

URL-wise the prefered solution is:
* `/players/[:id]/address` (note the singular)
* `/events/[:id]/address` (note the singular)
* `/teams/[:id]/addresses`

I guess this could be achieved by:
```ruby
resources :players do
  resource :address, shallow: true # note the singular
end
resources :events do
  resource :address, shallow: true # note the singular
end
resources :teams do
  resources :addresses, shallow: true
end
```

### PR DESCRIPTION
**Pull request by DenniJensen**
This Merge Request fixes #95. 
The routing problem from #95 will be handled in #96.

**IF this MR gets merged, pls run all migrations on production after auto deploy**

### CODE REVIEW

File: app/controllers/addresses_controller.rb [hunk 1]
```diff
@@ -18,4 +18,8 @@ def update
   def address_params
     params.require(:address).permit(:street, :zipcode, :city)
   end
+
+  def redirect_path_model
+    @address.addressable.player if @address.addressable
```
Comment (elgrillo83) [inline app/controllers/addresses_controller.rb]: As we are on Ruby 2.3 you could use the new Safe Navigation Operator here if you want, see: http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/

File: app/views/players/show.haml [hunk 2]
```diff
@@ -9,110 +9,29 @@
...
-            %dt= Player.human_attribute_name(:eligible_to_play_since)
-            %dd
-              = l(@player.eligible_to_play_since) if @player.eligible_to_play_since?
-
-          %dt= Player.human_attribute_name(:jersey_name)
-          %dd= @player.jersey_name
-
-          %dt= Player.human_attribute_name(:jersey_number)
-          %dd= @player.jersey_number
+    = render 'players/details', player: @player
```
Comment (elgrillo83) [inline app/views/players/show.haml]: the players prefix is not needed here

File: db/schema.rb [hunk 3]
```diff
@@ -87,32 +87,30 @@
...
+    t.datetime "datetime",                      null: false
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "description",   limit: 255
-    t.string   "address",       limit: 255,                 null: false
-    t.string   "name",                                      null: false
+    t.string   "description"
+    t.string   "address",                       null: false
+    t.string   "home_team"
+    t.string   "away_team"
```
Comment (elgrillo83) [inline db/schema.rb]: Are these columns new? 
`home_team:string` and `away_team:string`
What for?

**Comment by elgrillo83** [comment]
Nice work cleaning up some of the mess ;-)

File: app/views/players/show.haml (hunk 2 above)
Comment (DenniJensen) [inline app/views/players/show.haml]: I know, it looks weird with the absolut path to the template. I can remove it.

File: db/schema.rb (hunk 3 above)
Comment (DenniJensen) [inline db/schema.rb]: they are new, but not from me, I will check this

File: app/views/players/show.haml (hunk 2 above)
Comment (DenniJensen) [inline app/views/players/show.haml]: @elgrillo83  
it is this one
https://github.com/cfb-hasenheide/cfb-hasenheide/blob/refactor-database/db/migrate/20150110123631_add_team_fields_to_events.rb

File: db/schema.rb (hunk 3 above)
Comment (DenniJensen) [inline db/schema.rb]: After research, I think the `schema.rb` in master branch is wrong and the fields just missing

**Pull request merged**